archway/assetlist.json

```diff
@@ -43,6 +43,76 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
```

Comment:
Please register new assets to the bottom of the assets[] array instead of in the middle.